src/test/java/Homework19.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+    @Parameters({"BaseURL"})
+    @Test
+    public void deletePlaylist(String baseUrl) {
+        driver.get(baseUrl);
+        loginValideEmailPassowrd("momo.carlos5@gmail.com", "te$t$tudent");
+        clickPlayList();
+        clickDeletePlaylist();
+    }
+
+    private void clickDeletePlaylist() {
+        driver.findElement(By.cssSelector("[title='Delete this playlist']")).click();
+        WebElement notificationDeleted = driver.findElement(By.xpath("/html/body/div[2]/div"));
```

Comment:
But in this selector notificationDeleted, you are defining rather long and using Absolute xpath which is not best practices for locators.
//HINT: you can use [class='success show']